modules/health_quest/app/controllers/health_quest/v0/lighthouse_appointments_controller.rb

```diff
@@ -4,12 +4,23 @@ module HealthQuest
   module V0
     class LighthouseAppointmentsController < HealthQuest::V0::BaseController
       def index
-        render json: []
+        render json: factory.search(request.query_parameters).response[:body]
       end
 
       def show
         render json: {}
       end
+
+      private
+
+      def appointments_params
+        params.permit!
```

Comment:
Are we sure we want to permit all params here?

Reply:
> Are we sure we want to permit all params here?

Good catch! I was going to redo them separately in PR and handle the `show` action as well but I think I'll make the change here and now. Thank you!

Reply:
@ericboehs I've modified the allowed strong params. This will be used for the upcoming PR which implements the full functionality for the `show` action. The index action can accept anything from the `request.query_parameters` call; We'll be cherry-picking the parameters which will get used in the downstream modules and classes.

On a side note, I will be creating a PR or two down the line, when this feature is nearing code complete, which will perform security sweeps on the health-quest module and tie up any loose ends like the example above.